openapi/p4pa-pu-bff.openapi.yaml

```
@@ -74,6 +74,135 @@ paths:
+        '500':
+          description: Internal Server Error
+  /taxonomy/getMacroArea:
+    get:
+      tags:
+        - Taxonomy
+      operationId: getMacroArea
+      parameters:
+        - name: organizationType
+          in: query
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/hal+json:
+              schema:
+                  $ref: '#/components/schemas/CollectionModelTaxonomyMacroAreaCodeDTO'
+        '401':
+          description: Unauthorized - Missing or invalid token
+        '403':
+          description: Forbidden - Access denied
+        '500':
+          description: Internal Server Error
+  /taxonomy/getServiceType:
+    get:
+      tags:
+        - Taxonomy
+      operationId: getServiceType
+      parameters:
+        - name: organizationType
+          in: query
+          schema:
+            type: string
+        - name: macroAreaCode
+          in: query
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/hal+json:
+              schema:
+                $ref: '#/components/schemas/CollectionModelTaxonomyServiceTypeCodeDTO'
+        '401':
+          description: Unauthorized - Missing or invalid token
+        '403':
+          description: Forbidden - Access denied
+        '500':
+          description: Internal Server Error
+  /taxonomy/getCollectionReason:
+    get:
+      tags:
+        - Taxonomy
+      operationId: getCollectionReason
+      parameters:
+        - name: organizationType
+          in: query
+          schema:
+            type: string
+        - name: macroAreaCode
+          in: query
+          schema:
+            type: string
+        - name: serviceTypeCode
+          in: query
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/hal+json:
+              schema:
+                $ref: '#/components/schemas/CollectionModelTaxonomyCollectionReasonDTO'
+        '401':
+          description: Unauthorized - Missing or invalid token
+        '403':
+          description: Forbidden - Access denied
+        '500':
+          description: Internal Server Error
+  /taxonomy/getTaxonomyCode:
+    get:
+      tags:
+        - Taxonomy
+      operationId: getTaxonomyCode
+      parameters:
+        - name: organizationType
+          in: query
+          schema:
+            type: string
+        - name: macroAreaCode
+          in: query
+          schema:
+            type: string
+        - name: serviceTypeCode
+          in: query
+          schema:
+            type: string
+        - name: collectionReason
+          in: query
+          schema:
+            type: string
+      responses:
+        '200':
+          description: OK
+          content:
+            application/hal+json:
```

define directly the array here as done in:
https://github.com/pagopa/p4pa-auth/blob/develop/openapi/p4pa-auth.openapi.yaml#L184
```suggestion
            application/json:
```